Userland/Applications/PixelPaint/Filters/BoxBlur3.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2022, Tobias Christiansen <tobyase@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "Filter.h"
+
+namespace PixelPaint::Filters {
+
+class BoxBlur3 final : public Filter {
+public:
+    virtual void apply() const override;
+    static StringView filter_name() { return "Box Blur (3x3)"sv; }
```

Comment:
Maybe you could make this function virtual so that you wouldn't need to remember to set filter name in constructor?

Reply:
Yeah, I think I worked myself in a corner there and did not see the forest for the trees.
Thanks, fixed :^)